rust/main/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -939,10 +942,23 @@ impl Indexer<H256> for SealevelMailboxIndexer {
 #[async_trait]
 impl SequenceAwareIndexer<H256> for SealevelMailboxIndexer {
     async fn latest_sequence_count_and_tip(&self) -> ChainResult<(Option<u32>, u32)> {
-        // TODO: implement when sealevel scraper support is implemented
-        info!("Message delivery indexing not implemented");
+        let inbox = self
```

Comment:
might be a good idea to use / dedupe some of the existing logic we have for getting this here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/36a4c26a6fca91927e81456e16a7880894ab2259/rust/main/chains/hyperlane-sealevel/src/mailbox.rs#L454-L457, e.g. with a `get_inbox_account` or something

Reply:
I extracted common part into a method.